app/models/samples_workflow_execution.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# entity class for Sample
+class SamplesWorkflowExecution < ApplicationRecord
+  has_logidze
+  acts_as_paranoid
+
+  belongs_to :workflow_execution
+  belongs_to :sample
+
+  serialize :samplesheet_params, coder: JsonbIndifferentSerializer
```

Comment:
I don't think we need to add a serialize section here, by default jsonb will be handled and persisted correctly, If you look at the `Sample` model we don't have a `serialize` section and it correctly persists the `metadata` attribute and the only difference is you can't access values with `:key` you need to use `'key'` instead. This also applies to the `Attachment` model where we also have a `metadata` jsonb column.

Reply:
fixed in https://github.com/phac-nml/irida-next/pull/308/commits/b93ceec0d2aa3386add4aafbf34c30d56610ee7f